posthog/api/test/test_trends.py

```diff
@@ -101,7 +101,7 @@ def test_includes_only_intervals_within_range(client: Client):
                     "math": "dau",
                     "math_property": None,
                     "math_group_type_index": None,
-                    "properties": [],
+                    "properties": {},
```

Comment:
q: why did we have to do this? 🤔

Reply:
hmm? the to_dict always returns a dict now and not a list